specificatie/openapi.yaml

```diff
@@ -16,33 +16,33 @@ paths:
     get:
       operationId: ingeschrevenNatuurlijkPersonen
       description: |
-        Het ophalen van een collectie ingeschreven personen inclusief het verblijfsadres, ouders, partners, kinderen en reisdocumenten. Het betreft alleen actuele gegevens van de betreffende ingeschreven personen. Ten minste één van de volgende combinaties van parameters moet zijn opgenomen:
+        Het ophalen van een collectie ingeschreven personen inclusief het verblijfsadres, ouders, partners, kinderen en reisdocumenten. Het betreft alleen actuele gegevens van de betreffende ingeschreven personen. Ten minste een van de volgende combinaties van parameters moet zijn opgenomen:
 
 
         1.  Persoon
-            -  geboorte__datum
```

Comment:
deze wijzigingen zijn niet ook doorgevoerd in de genereervariant, terwijl die wel met wijzigingen in deze pull request zit. Hopelijk komt dat automatisch goed, want genereervariant hoort nu bij commit automatisch te worden gemaakt.

Reply:
Dat komt automatisch goed. Inmiddels ben ik erachter dat het opnemen van wijzigingen in de genereervariant in de pull request overbodig is. Dat zal ik vanaf nu ook niet meer doen. De genereervariant wordt opnieuw gresolved op het moment dat deze pull-request wordt gemerged.